### PR TITLE
[ASM] - EXPANDER - 10863 - Fix for Cortex ASM - Azure Enrichment Playbook

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[EXPANDR-10863](https://jira-dc.paloaltonetworks.com/browse/EXPANDR-10863)

## Description
Fixes an issue in the playbook where Azure Hierarchy Hierarchy task's default path was not handled correctly.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by BigEasyJ** [comment]
Please review @capanw @johnnywilkes

**Review by johnnywilkes — APPROVED** [review]
@BigEasyJ , thanks for the quick response, Ship it!

**Comment by BigEasyJ** [comment]
@rshunim @melamedbn this ready to go

**Comment by rshunim** [comment]
@melamedbn @sashakulakov who is reviewing this ticket?

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , please review

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_47.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex ASM - Azure Enrichment
+
+Fixed an issue in the playbook where Azure Hierarchy Hierarchy task's default path was not handled correctly.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_47.md:6]:
```suggestion
Fixed an issue in the playbook where the Azure Hierarchy Hierarchy task's default path was not handled correctly.
```

**Comment by ShirleyDenkberg** [comment]
@johnnywilkes @melamedbn @rshunim Doc review completed.

**Review by ssokolovich — COMMENTED** [review]
@BigEasyJ  - please update the PB image as well. 
@johnnywilkes - please mention @rshunim once the PR is ready to be merged :)

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_47.md (hunk 1 above)
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_47.md]: Thank you for the suggestion. Updated release notes.

**Comment by capanw** [comment]
@ssokolovich Added the new PB image.

@rshunim, made recommended changes and this is ready to be merged. Thank you!!!

**Review by rshunim — APPROVED** [review]
Great Job!

**Pull request merged**